external/model-preparation-algorithm/mpa_tasks/extensions/datasets/mpa_cls_dataset.py

```diff
@@ -128,7 +128,12 @@ def evaluate(self,
             metrics.remove('class_accuracy')
             self.class_acc = True
 
+        # compute top-k metrics from mmcls and align them in [0,1] range
         eval_results = super().evaluate(results, metrics, metric_options, logger)
+        for k in metric_options['topk']:
+            if 0 <= eval_results[f'accuracy_top-{k}'] <= 100:
```

Comment:
top-k score is always <= 100. Isn't it?

Reply:
I'm wrong. [0, 1] included in [0, 100]. We should check case [ 1, inf)